INSTALL.md

```diff
@@ -93,6 +93,44 @@ $ make -f policy_utilities.mak clean
 $ make -f policy_utilities.mak
 ```
 
+If you wish to use the Policy Generator instead of manually composing the
```

Comment:
Here, do we need to clarify that " ... use the Policy Generator for building the `simple_app_under_oe` ...".

Am not sure if this generator will be used for / by other apps. So, it's clearer to narrow its usage to the one app that needs it.

Reply:
Policy Generator works with any backend. Not just OE. For instance, SEV-SNP sample uses it too.